uint8 GEOFENCE_MAX_VERTICES = 16

uint32 count    			# number of actual vertices
fence_vertex[16] vertices		# geofence positions
